examples/mamba/mamba/trainer.py

```diff
@@ -0,0 +1,72 @@
+from typing import Optional, Type, Union
+
+from torch.nn.parallel import DistributedDataParallel
+
+from nanotron import logging
+from nanotron.config import Config
+from nanotron.trainer import DistributedTrainer
+
+logger = logging.get_logger(__name__)
+
+from nanotron import distributed as dist
+from nanotron.config import Config, ExistingCheckpointInit, MambaInit
+from nanotron.logging import log_rank
+from nanotron.models import NanotronModel
+from nanotron.parallel.tied_parameters import get_tied_id_to_param
+from nanotron.serialize import load_weights, parse_ckpt_path
+
+
+class MambaTrainer(DistributedTrainer):
+    def __init__(
```

Comment:
this looks the same as https://github.com/huggingface/nanotron/blob/9f117eeb1e8309c45e2b1820069292ed5d7c4647/src/nanotron/trainer.py#L107-L113

duplication?

Reply:
it looks exactly like https://github.com/huggingface/nanotron/blob/9f117eeb1e8309c45e2b1820069292ed5d7c4647/src/nanotron/trainer.py#L107-L113, we could remove it from MambaTrainer right?